static_test_driver/static_test_driver.ino

```diff
@@ -207,7 +214,6 @@ void loop() {
       for (int i = 0; i < NUM_HIST_VALS; i++)
```

Comment:
To clarify: READ_FLAG(zero) should still be there, it should just have a call to tare() instead of what was there.